src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -780,4 +782,10 @@ public static boolean getMobGriefingEvent(World world, Entity entity)
         Result result = event.getResult();
         return result == Result.DEFAULT ? world.getGameRules().getBoolean("mobGriefing") : result == Result.ALLOW;
     }
+
+	public static BufferedImage getResourceBufferImage(String s) {
```

Comment:
Need to fix formatting here

Reply:
Dag nab it, I thought I checked it all. Thanks :)